src/System.Runtime/tests/System/ArrayOfPointers.ArrayTests.cs

```diff
@@ -0,0 +1,68 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Xunit;
+
+namespace System.Tests
+{
+    public static partial class ArrayTests
```

Comment:
This one is not temporarily. This also fails the compilation, but it fails saying that Array of pointers are not supported on PN, which is why I don't think these tests will ever run on uapaot.